translations/fr.yaml

```diff
@@ -229,23 +229,6 @@ ember-gwr:
 
     logout: Se déconnecter
 
-    projectForm:
-      save: Enregistrer
-      cancelImport: Interrompre l'import
-      applyAll: Tout reprendre
-      importFromCaluma: Importer les données de la demande de permis
-      changeStatus: Changer le statut
-      dates: Dates
-      datePlaceholder: Choisissez une date...
-      title: Projet de construction {id}
-      realEstateHeading: Identification de l'immeuble (bien-fonds)
-      clientHeading: Maître d’ouvrage
-
-      diff:
-        existingData: Données existantes dans le RegBL
-        importedData: Données importées
-        apply: Reprendre
-
     modelForm:
```

Comment:
Missing
```
modelForm:
    noImportData: Il n'y a pas de données à importer.
```

(from deepl)

Reply:
We dont add french translations yet. We have to get them from an official translation company from kt bern.